test/test-arch-package.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+SRC_DIR=$1
+
+if [[ "x${SRC_DIR}" == "x" ]]; then
+	echo "Usage:"
+	echo "  test-arch-package <project_src_dir>"
+	exit 1
+fi
+
+cd $SRC_DIR
+
+sudo docker build -t pac4cli -f archlinux/Dockerfile .
```

Comment:
On debian, it's possible to do `addgroup $USER docker` and run `docker` without `sudo` after that. I have a mild preference for not having `sudo` in the script, but I won't insist if that makes it unergonomic on your setup. What do you think?

Reply:
Oh, and can we put `archlinux` somewhere in the tag for the image?

Reply:
Make sense. I merged the branch for now (not to block others) and will do the requested changes soon